doc/api/fs.md

```diff
@@ -1631,6 +1649,11 @@ then the file descriptor won't be closed, even if there's an error.
 It is the application's responsibility to close it and make sure there's no
 file descriptor leak.
 
+By providing the `open`, `write`, `writev` and `close` options it is possible
+to override the used corresponding `fs` implementation. Overriding `write()`
+without `writev()` can reduce performance as some optimizations (`_writev()`)
+will be disabled.
```

Comment:
what is the signature for `writev`?

Reply:
It's documented under `fs.writev()`, just like the rest.

Reply:
Thanks to https://github.com/nodejs/node/commit/e4bbbcc84bd10be1e9e8e66d42542c70c1a02056